analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -99,17 +99,22 @@ class Gradle(
                 "${artifact.groupId}/${artifact.artifactId}/${artifact.version}"
             )
 
-            val artifactFile = artifactRootDir.walk().find {
+            val artifactFiles = artifactRootDir.walk().filter {
```

Comment:
Could you add to the commit message or as a code comment why the most recent edited file is chosen / is better than an arbitrary one? I assume it is because it is most likely the one downloaded during the current dependency resolution?

Reply:
I've added a code comment.